modules/gsp-cluster/main.tf

```diff
@@ -204,6 +131,9 @@ module "kiam-system" {
     kiam:
       server:
         assumeRoleArn: "${aws_iam_role.kiam_server_role.arn}"
+      agent:
+        host:
+          interface: "eni+"
```

Comment:
I did some investigation around the regex match ... this does appear to be the recommendation from both kube2iam and kiam (both which use the iptables match) based on other peoples configurations and issues... I'm still a bit unsure about the relationship between Node level networking and Pod level networking in the land of VPC CNI but I'm confident this should work as expected 👍